src/index.js

```diff
@@ -11,6 +11,26 @@ import Track from './api/track';
 import SDK_VERSION from './version';
 import STATUS from './status';
 
+const handleError = (callback) => {
+  return (err) => {
+
+    // Radar Error
```

Comment:
nit: can we mention the JS SDK explicitly throwing these? slightly confusing as an http error is still one the Radar backend returns

Reply:
We're still explicitly throwing all of them. The only difference is we need to pass back an object for the `http` errors so we can capture the response, so we handle those separately.